test/notifications/display_test.dart

```diff
@@ -955,6 +963,68 @@ void main() {
       receiveFcmMessage(async, removeFcmMessage([message2], account: account2));
       check(testBinding.androidNotificationHost.activeNotifications).isEmpty();
     })));
+
+    test('removeNotificationsForAccount: removes notifications', () => runWithHttpClient(() => awaitFakeAsync((async) async {
+      await init();
+      final message = eg.dmMessage(from: eg.otherUser, to: [eg.selfUser]);
+
+      receiveFcmMessage(async, messageFcmMessage(message));
+      check(testBinding.androidNotificationHost.activeNotifications).isNotEmpty();
+      await NotificationDisplayManager.removeNotificationsForAccount(
```

Comment:
nit: while joining checks to the steps that set them up, separate from the steps that come after them (which set up later checks):
```suggestion
      check(testBinding.androidNotificationHost.activeNotifications).isNotEmpty();

      await NotificationDisplayManager.removeNotificationsForAccount(
```

See related comment above. (There are also other examples below.)